src/main/java/Data.java

```diff
@@ -0,0 +1,84 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.IOException;
```

Comment:
Java coding standard: The ordering of import statements must be consistent. Consider grouping the `io` together and the `util` together.